### PR TITLE
55 upload demo datasets

### PR DESCRIPTION
**Pull request by Sorrento110**
Merge in the changes to `make populate` in order to store mock dataset metadata including features and qualifiers relevant to the datasets.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,14 +18,14 @@ tidy:
 
 .PHONY:
 up:
-	docker compose up --build -d;
+	docker-compose up --build -d;
```
Comment (fivegrant) [inline Makefile:21]: We can ensure people are using v2 of docker compose if we leave out the dash: https://www.docker.com/blog/announcing-compose-v2-general-availability/

something to think about

**Comment by fivegrant** [comment]
adding @marshHawk4 as a reviewer because he created the initial script

File: Makefile (hunk 1 above)
Comment (Sorrento110) [reply Makefile]: I have fixed this in the newest push.

**Comment by Sorrento110** [comment]
@marshHawk4 @fivegrant Do you guys use the container names for anything? The last push I made to this branch explicitly names them so I wanted to make sure there was awareness that that might be a breaking change.

File: scripts/upload_demo_data.py [hunk 2]
```diff
@@ -239,5 +240,7 @@ def asset_to_project(project_id, asset_id, asset_type):
     except Exception as e:
         print(e)
 
+programatically_populate_datasets()
```
Comment (fivegrant) [inline scripts/upload_demo_data.py:243]: how do we handle the case where the user doesn't have data annotation?

**Comment by fivegrant** [comment]
> @marshHawk4 @fivegrant Do you guys use the container names for anything? The last push I made to this branch explicitly names them so I wanted to make sure there was awareness that that might be a breaking change.

sounds fine to me @Sorrento110

File: scripts/upload_demo_data.py (hunk 2 above)
Comment (Sorrento110) [reply scripts/upload_demo_data.py:243]: This script is independent of the data-annotation stack, it doesn't make any calls to the data-annotation API.

**Review by mattprintz — APPROVED** [review]

File: scripts/upload_demo_data.py (hunk 2 above)
Comment (Sorrento110) [reply scripts/upload_demo_data.py:243]: It's dependent on experiments directory though, so if the BIOMD sim-output.json files change it will break.

**Review by marshHawk4 — APPROVED** [review]
Looks good.

**Pull request merged**